bfvmm/include/exit_handler/exit_handler_intel_x64.h

```diff
@@ -135,16 +145,23 @@ class exit_handler_intel_x64
     virtual void handle_xsaves();
     virtual void handle_xrstors();
 
-    virtual void advance_rip();
-    virtual void unimplemented_handler();
+    virtual void advance_rip() noexcept;
+    virtual void unimplemented_handler() noexcept;
```

Comment:
I can understand noexcept for advance_rip, but is there not a case where unimplemented would/should throw?